docs/data/sdks/android-kotlin/index.md

```diff
@@ -293,3 +293,494 @@ amplitude.add(
+| `disableDeviceModel()` | Disable tracking of device model |
+| `disableDma()` | Disable tracking of user's designated market area (DMA). |
+| `disableIpAddress()` | Disable tracking of user's IP address |
+| `disableLanguage()` | Disable tracking of device's language |
+| `disableLatLng()` | Disable tracking of user's current latitude and longitude coordinates |
+| `disableOsName()` | Disable tracking of device's OS Name |
+| `disableOsVersion()` | Disable tracking of device's OS Version |
+| `disablePlatform()` | Disable tracking of device's platform |
+| `disableRegion()` | Disable tracking of user's region. |
+| `disableVersionName()` | Disable tracking of your app's version name |
+
+!!!note
+
+    The `TrackingOptions` only prevents default properties from being tracked on newly created projects, where data has not yet been sent. If you have a project with existing data that you would like to stop collecting the default properties for, please get help in the [Amplitude Community](https://community.amplitude.com/). Note that the existing data **is not** deleted.
+
+### COPPA control
+
+COPPA (Children's Online Privacy Protection Act) restrictions on IDFA, IDFV, city, IP address and location tracking can all be enabled or disabled at one time. Apps that ask for information from children under 13 years of age must comply with COPPA.
+
+=== "Java"
+
+    ```java
+    amplitude = AmplitudeKt.Amplitude(BuildConfig.AMPLITUDE_API_KEY, getApplicationContext(), configuration -> {
+        configuration.setEnableCoppaControl(true); //Disables ADID, city, IP, and location tracking
+        return Unit.INSTANCE;
+    });
+    client.enableCoppaControl();
+    ```
+
+=== "Kotlin"
+
+    ```kotlin
+    amplitude = Amplitude(
+        Configuration(
+            apiKey = AMPLITUDE_API_KEY,
+            context = applicationContext,
+            enableCoppaControl = true //Disables ADID, city, IP, and location tracking
+        )
+    )
+    ```
+
+### Advertiser ID
+
+Advertiser ID (also referred to as IDFA) is a unique identifier provided by the iOS and Google Play stores. As it's unique to every person and not just their devices, it's useful for mobile attribution.
+ [Mobile attribution](https://www.adjust.com/blog/mobile-ad-attribution-introduction-for-beginners/) is the attribution of an installation of a mobile app to its original source (such as ad campaign, app store search).
+ Mobile apps need permission to ask for IDFA, and apps targeted to children can't track at all. Consider IDFV or device ID when IDFA isn't available.
+
+Follow these steps to use Android Ad ID.
+
+!!!warning "Google Ad ID and Tracking Warning"
+
+    As of April 1, 2022, Google allows users to opt out of Ad ID tracking. Ad ID may return null or error. You can use am alternative ID called [App Set ID](#app-set-id), which is unique to every app install on a device. [Learn more](https://support.google.com/googleplay/android-developer/answer/6048248?hl=en)
```

Comment:
Indent: this whole text block must be indented 4 spaces for the warning block to work. Double-check in your editor, please.

Reply:
It looks right in my editor and on the web. 
I think it's the github issue.
<img width="1196" alt="Screen Shot 2022-10-07 at 10 00 20 AM" src="https://user-images.githubusercontent.com/13028329/194611318-bdbd9280-5cfb-4e79-932d-aa3c091f13a5.png">
<img width="770" alt="Screen Shot 2022-10-07 at 10 17 15 AM" src="https://user-images.githubusercontent.com/13028329/194613951-e94e97e1-f855-4fa1-bae3-9712d56d130b.png">

Reply:
Thank you so much for confirming, sorry for the trouble!